include/GFp/base.h

```diff
@@ -56,6 +56,10 @@
 
 // This file should be the first included by all BoringSSL headers.
 
+#if defined(BORINGSSL_PREFIX)
+#include <boringssl_prefix_symbols.h>
```

Comment:
@briansmith: I am including the symbol headers in `base.h`. I'm confused about what else you'd like me to include in this file...